src/vcpkg-fuzz/main.cpp

```diff
@@ -68,14 +68,16 @@ namespace
                     }
                     else
                     {
-                        print2(Color::error, "Invalid kind: ", value, "\n");
-                        print2(Color::error, "  Expected one of: utf-8, json, platform-expr\n\n");
+                        msg::print_error(LocalizedString::from_raw(fmt::format("invalid kind: {}", value))
+                                             .appendnl()
+                                             .append_indent()
+                                             .append_raw("expected one of: utf-8, json, platform-expr"));
                         print_help_and_exit(true);
```

Comment:
If we're going to print localized help here, should the expected message also be localized?

Reply:
Agreed, this should either use `msg::write_unlocalized_text_to_stdout` or it should be localized.